src/workerd/jsg/jsg.h

```diff
@@ -2365,6 +2365,15 @@ class Lock {
     return Ref<T>(kj::refcounted<T>(kj::fwd<Params>(params)...));
   }
 
+  // Returns a kj::String with an external memory adjustment attached.
+  kj::String accountedKjString(kj::Array<char>&& str);
+  kj::String accountedKjString(kj::String&& str) {
+    return accountedKjString(str.releaseArray());
+  }
+  kj::String accountedKjString(kj::StringPtr str) {
```

Comment:
Before we go with more refactorings, I strongly think we should drop "accounted" from this. 

I feel like Sean Parker.

Reply:
The naming of these is far less important than making these improvements. If we decide that a rename is necessary then it'll be super trivial to search for and find these all at once.